controllers/topics.controllers.js

```diff
@@ -0,0 +1,12 @@
+const {retrieveTopics} = require("../models/topics.models.js")
+const topicData = require("../db/data/test-data/index.js")
+
+exports.getTopics = (request, response) => {
+    retrieveTopics(topicData).then((topic)=>{
```

Comment:
since this is an array of all the topics you're getting back, topics would be a better variable name than just topic